allennlp/training/metrics.py

```diff
@@ -288,3 +290,196 @@ def reset(self):
         self._true_negatives = 0.0
         self._false_positives = 0.0
         self._false_negatives = 0.0
+
+
+@Metric.register("conll_span_f1")
+class ConllSpanBasedF1Measure(Metric):
```

Comment:
I'd remove `Conll` from this name - Cristian could use this for his F1 metric, for instance.  It's just a general span-based F1 for BIO-tagged data.